openpype/settings/entities/schemas/projects_schema/schema_project_fusion.json

```diff
@@ -46,6 +46,13 @@
                 }
             ]
         },
+        {
+            "type": "text",
+            "multiline" : true,
+            "use_label_wrap": true,
+            "key": "masterprefs",
+            "label": "Fusion Master Preferences"
+         },
```

Comment:
@iLLiCiTiT is there any way to increase the size of the text input field in the settings UI?

![afbeelding](https://user-images.githubusercontent.com/2439881/229770412-53900edb-1028-4f1f-ad0b-e1b13b16b978.png)

This feels very cramped.

Reply:
Not at this moment. Please create an issue, I may look into that during weekend.

Reply:
Created https://github.com/ynput/OpenPype/issues/4774